pmm-app/src/pmm-inventory/panel.constants.tsx

```diff
@@ -1,52 +1,51 @@
 import React from 'react';
 
-const getCustomLabels = customLabels =>
+export const getCustomLabels = customLabels =>
   customLabels.map(label => <span key={label.key}>{`${label.key}: ${label.value}`}</span>);
 
-const mainColumns = ['service_id', 'type', 'service_name', 'custom_labels', 'node_id', 'address', 'port'];
+export const MAIN_COLUMN = [
+  'service_id',
+  'type',
+  'service_name',
+  'custom_labels',
+  'node_id',
+  'address',
+  'port',
+];
 
-// TODO(lunaticusgreen): extract all `render`s to separate components
-export const servicesColumns = [
+export const SERVICES_COLUMNS = [
   {
-    title: 'ID',
-    dataIndex: 'service_id',
-    key: 'service_id',
+    Header: 'ID',
+    accessor: 'service_id',
   },
   {
-    title: 'Service Type',
-    dataIndex: 'type',
-    key: 'type',
+    Header: 'Service Type',
+    accessor: 'type',
   },
   {
-    title: 'Service name',
-    dataIndex: 'service_name',
-    key: 'service_name',
+    Header: 'Service name',
+    accessor: 'service_name',
   },
   {
-    title: 'Node ID',
-    dataIndex: 'node_id',
-    key: 'node_id',
+    Header: 'Node ID',
+    accessor: 'node_id',
   },
   {
-    title: 'Addresses',
-    dataIndex: 'address',
-    key: 'address',
+    Header: 'Addresses',
+    accessor: 'address',
   },
   {
-    title: 'Port',
-    dataIndex: 'port',
-    key: 'port',
+    Header: 'Port',
+    accessor: 'port',
   },
   {
-    title: 'Other Details',
-    dataIndex: 'age',
-    key: 'age',
-    render: (text, element) => {
-      const labels = Object.keys(element).filter(label => !mainColumns.includes(label));
+    Header: 'Other Details',
+    accessor: element => {
```

Comment:
This is a separate component. If we save it to a separate file, we have a chance to write a test.